README.md

```diff
@@ -65,13 +65,23 @@ export const parameters = {
 };
 ```
 
-When using Web Components, the HTML will contain empty comments, i.e. `<!---->`.
-If you want to remove these, use the `removeEmptyComments` parameter:
+When using Web Components, the HTML will contain empty comments, i.e. `<!---->`, or `<!--..lit$..-->` if using [lit-html](https://lit.dev/docs/). If you want to remove these, use the `removeComments` parameter:
 
 ```js
 export const parameters = {
   html: {
-    removeEmptyComments: true, // default: false
+    removeComments: true, // default: false
```

Comment:
Please leave `removeEmptyComments` in the documentation since it's still there in the code (which it should be so that it's backwards compatible)